pkg/schedule/labeler/rules.go

```diff
@@ -38,6 +39,10 @@ type RegionLabel struct {
 	expire  *time.Time
 }
 
+func (l *RegionLabel) String() string {
+	return fmt.Sprintf("key: %s, value: %s", l.Key, l.Value)
```

Comment:
Does it need to be `{}` as well?
```suggestion
	return fmt.Sprintf("key: {%s}, value: {%s}", l.Key, l.Value)
```

Reply:
No

Reply:
got it